File: 389-find-the-difference/389-find-the-difference.py
# https://youtu.be/sRwElQ_TOr8

class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        c = 0
        
        for i in s:
            c = c ^ ord(i)
        
        for j in t:
            c = c ^ ord(j)
        
        return chr(c)

File: 389-find-the-difference/389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) 
    {
        char c = 0;
        
        for(char i: s)
            c = c ^ i;
        
        for(char j: t)
            c = c ^ j;
        
        return c;  
    }
};